src/Microsoft.AspNetCore.Hosting/Internal/StartupLoader.cs

```diff
@@ -83,11 +113,17 @@ private static ConfigureBuilder FindConfigureDelegate(Type startupType, string e
             return new ConfigureBuilder(configureMethod);
         }
 
+        private static ConfigureContainerBuilder FindConfigureContainerDelegate(Type startupType, string environmentName)
+        {
+            var configureMethod = FindMethod(startupType, "Configure{0}Container", environmentName, typeof(void), required: false);
```

Comment:
Wait... Scratch that. I see you've made the `Invoke` method a no-op when `MethodInfo` is null.